app/controllers/org_admin/templates_controller.rb

```diff
@@ -178,33 +166,39 @@ def history
       @current_tab = params[:r] || 'all-templates'
     end
     
-    # GET /org_admin/templates/:id/customize
+    # POST /org_admin/templates/:id/customize
     # -----------------------------------------------------
     def customize
       @template = Template.find(params[:id])
       authorize @template
+      @current_tab = 'funder-templates'
       # TODO use POST instead of GET since we are effectively adding a new template resource
-      # TODO add check @template.customize? before
-      customisation = @template.customize!(current_user.org)
-      
-      @current_tab = params[:r] || 'all-templates'
-      redirect_to edit_org_admin_template_path(customisation, r: 'funder-templates')
+      begin
+        customisation = @template.customize!(current_user.org)
+        redirect_to edit_org_admin_template_path(customisation, r: @current_tab)
+      rescue Exception => e
+        flash[:alert] = _('Unable to customize that template.')
+        redirect_to org_admin_templates_path(r: @current_tab)
+      end
```

Comment:
Please, consider this change:
```
if customize?(current_user.org)
  begin
    customisation = @template.customize!(current_user.org)
    redirect_to edit_org_admin_template_path(customisation, r: @current_tab)
  rescue => e
    flash[:alert] = _('Unable to customize that template.')
    redirect_to org_admin_templates_path(r: @current_tab) 
  end
else
  redirect_to ... #perhaps to index ?
end
```
and remember that we want to rescue from exceptions that are recoverable, by default StandError or any subclass and in our case:
- ActiveRecord::StatementInvalid (not yet in code but will be raised when family_id and version already exists)
- ActiveRecord::RecordInvalid (if any validation for template fails)
- RuntimeError (the raise clauses we have right at the top of the method)